public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -30,6 +30,27 @@ function Task (geojson, tutorialTask, currentLat, currentLng, startPointReversed
         tutorialTask: tutorialTask
     };
 
+    // Observed area radius.
```

Comment:
Feel free to add the comments to the same line as the variable to save some vertical space